mix.exs

```diff
@@ -76,7 +76,7 @@ defmodule EctoSQL.MixProject do
     if path = System.get_env("ECTO_PATH") do
       {:ecto, path: path}
     else
-      {:ecto, github: "elixir-ecto/ecto"}
+      {:ecto, github: "zachdaniel/ecto", branch: "subqueries-in-order-by"}
```

Comment:
Please update this to point to main, address the other feedback, and we should be good. :)